styles.css

```diff
@@ -107,22 +107,22 @@ blockquote:hover {
 /*********** HEADER ***********/
 /******************************/
 
-.header__container {
+.heading__container {
   display: flex;
   align-items: center;
   justify-content: space-between;
   margin-bottom: 3.5rem;
 }
 
-.header__title {
+.heading__title {
   display: flex;
   align-items: center;
   gap: 1rem;
   color: var(--clr-gray-6);
   font-weight: 800;
```

Comment:
For the sake of understanding how these reviews work, I'm just going to request changing the font-weight to something a little less bold.